header/interface.go

```diff
@@ -60,12 +60,18 @@ type Exchange interface {
 	Getter
 }
 
+// RequestSize defines the max amount of headers that can be requested/handled at once.
+var RequestSize uint64 = 512
```

Comment:
I see the motivation here. To make it reusable from the HeaderEx and Syncer but we should not pull protocol-level specifics to the interfaces. Instead, it should be `maxRequestSize` const in the HeaderEx and the `requestSize` global var in the Syncer the way it was. That is, these parameters are two separate things. One is the limit, the other is the size.

Reply:
agree